public/app/features/dashboard/components/DashboardSettings/SettingsCtrl.ts

```diff
@@ -121,8 +120,12 @@ export class SettingsCtrl {
     const url = this.$location.path();
 
     for (const section of this.sections) {
-      const sectionParams = _.defaults({ editview: section.id }, params);
-      section.url = getConfig().appSubUrl + url + '?' + $.param(sectionParams);
+      section.url =
+        getConfig().appSubUrl +
+        urlUtil.renderUrl(url, {
+          ...params,
+          editview: section.id,
+        });
```

Comment:
Maybe you could use `locationUtil.assureBaseUrl` for this?

Reply:
yes, Updated!